third_party/libusb/naclport/src/chrome_usb/chrome-usb-backend.js

```diff
@@ -222,12 +243,15 @@ ChromeUsbBackend.prototype.reportRequestError_ = function(
 
 /**
  * @param {string} debugRepresentation
+ * @param {string} functionName API function that was called
  * @param {!goog.promise.Resolver} promiseResolver
  * @param {!Array} resultArgs
  * @private
  */
 ChromeUsbBackend.prototype.reportRequestSuccess_ = function(
-    debugRepresentation, promiseResolver, resultArgs) {
+    debugRepresentation, functionName, promiseResolver, resultArgs) {
+  this.requestSuccessHooks_.forEach(function f(hook, key, map) {
```

Comment:
nit: The name "f" is not informative. Either make it unnamed, or use a lambda, or maybe just have a loop.